app/static/css/style.css

```diff
@@ -284,3 +403,66 @@ footer {
     position: absolute;
   }
 }
+
+
+
+/**
+ * Team Photo
+ */
+
+.team-photo {
+  border-radius: 20px;
+  position: relative;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+  height: 300px;
+  overflow: hidden;
+}
+
+.team-photo:hover > .team-name {
+  visibility: visible;
+  top: auto;
+  bottom: 10px;
+}
+
+.team-photo._1 {
+  background-image: url('/static/img/team/stevengong.png');
+}
+
+.team-photo._2 {
+  background-image: url('/static/img/team/louisfy.jpeg');
+}
+
+.team-photo._3 {
+  background-image: url('/static/img/team/ishita.jpg');
+}
+
+.team-photo._4 {
+  background-image: url('/static/img/team/stevenguikal.jpg');
+}
+
+.team-photo._5 {
+  background-image: url('/static/img/team/justinlu.jpg');
+}
+
+.team-photo._6 {
+  background-image: url('/static/img/team/aavasapkota.jpg');
+}
+
+.team-photo._7 {
+  background-image: url('/static/img/team/celesteluo.jpg');
+}
```

Comment:
This kind of stuff is better for in lining and it allows using `{{ static('...') }}` tags.